data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Try and find another way in.)`
+					goto attrr
+				`	(Continue trying to bust the gate open.)`
+			`	You keep on pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate, taking turns with others when they get tired. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate, which takes some good damage with each time, gradually budging more. It cannot hit enough times, sadly, as the whistle from the start of the event sounds again, meaning that the time has run out, and that the defenders have won.`
+				goto lost
+			label attrr
+			`	You and a part of the group stick close to the walls, and follow them until the other side, where you find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way, before charging at it and pushing, nearly forcing it open. More defenders come to hold it, though, as it becomes increasingly difficult to budge it. One Arach heads back to the main group to call for more to help, and when they arrive so do more defenders. The struggle continues for a while, before loud bangs are heard from the other side of the fort, and the Kimek organizer comes around the back to tell everyone to return to the main entrance. As you come to the main gate, you see the battering ram has forced the main gate open, something that must've been made easier with so many leaving it unguarded to instead defend the back gate. With that, the whistle sounds again, marking the end of the battle, and announcing that the attackers have won.`
+			label won
+			`	The Kimek organizer in charge of your side starts a cheer, which quickly spreads to the others. After the leaders of both teams meet to greet each other, everyone is asked to return the colored cloth they were given, and make their way down the hill. Down by the camp, the Kimek who brought you here is waiting to guide your group to many short tables with plates on them; apparently, the reward for winning is a feast. As most of those who participate are Kimek, almost all of the food comes the way they prefer it, a soupy, half-digested mess, but thankfully there are a few meals prepared the way the other species like them, meaning you can pick one prepared for Saryds, which is as close as you could get to human cuisine here.`
```

Comment:
With all of the slippery liquid involved and the mud that would probably produce, and especially considering that Saryd have hair and a fur coat, and likely Arach have fine hairs for helping sense their environment, it might make sense that there's some kind of communal washing station to clean the worst of the muck off before they get around to trying to eat food.